l10n_co_geo/data/res_config_co.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <data noupdate="1">
+    <!-- Install es_CO language -->
+    <record model="base.language.install" id="install_co">
```

Comment:
@kurkop entiendes la retroalimentación de Pedro? Esto significa que no podemos utilizar este modulo clandestinamente en los desplieges para este fin.
@pedrobaeza ok, this needs some refactoring of our deployment strategy, but you're absolutely right.